README.md

```diff
@@ -14,11 +14,11 @@ import { getPage } from 'next-page-tester';
 
 describe('Blog page', () => {
   it('renders blog page', async () => {
-    const Page = await getPage({
+    const { page } = await getPage({
```

Comment:
Hi @Meemaw,
quick help request :)

How do you would be a good name for the property holding the returned page element? Consider that it might be a page element or the `App` element wrapping the page.

Candidates:
- page
- element
- pageElement
- anything else?

Reply:
Hmmm, I think I would stay with `page` for now. Anyway those are always hard to get right the first time so I wouldn't worry too much about it. I think with usage and adoption cleaner API will emerge if needed.

Btw, unrelated question regarding the `App` element. If I understand correctly it is not included by default. How come?

Reply:
Let's stick to `page`, then. Thanks 🙌 

I set the app option to false by default just because I wanted to be defensive. I'll default it to true with v.0.8.0.